internal/filer_test.go

```diff
@@ -462,3 +473,281 @@ func TestAccFilerWorkspaceNotebookWithOverwriteFlag(t *testing.T) {
 	filerTest{t, f}.assertContents(ctx, "scalaNb", "// Databricks notebook source\n println(\"second upload\"))")
 	filerTest{t, f}.assertContents(ctx, "jupyterNb", "# Databricks notebook source\nprint(\"Jupyter Notebook Version 2\")")
 }
+
+func TestAccFilerWorkspaceFuseReadDir(t *testing.T) {
+	files := []struct {
+		name    string
+		content string
+	}{
+		{"pyNb.py", "# Databricks notebook source\nprint('first upload'))"},
+		{"rNb.r", "# Databricks notebook source\nprint('first upload'))"},
+		{"sqlNb.sql", "-- Databricks notebook source\n SELECT \"first upload\""},
+		{"scalaNb.scala", "// Databricks notebook source\n println(\"first upload\"))"},
+		{"jupyterNb.ipynb", jupyterNotebookContent1},
+		{"jupyterNb2.ipynb", jupyterNotebookContent2},
+		{"dir1/dir2/dir3/file.txt", "file content"},
+		{"foo.py", "print('foo')"},
+		{"foo.r", "print('foo')"},
+		{"foo.sql", "SELECT 'foo'"},
+		{"foo.scala", "println('foo')"},
+	}
+
+	ctx := context.Background()
+	wf, _ := setupWsfsFuseFiler(t)
+
+	for _, f := range files {
+		err := wf.Write(ctx, f.name, strings.NewReader(f.content), filer.CreateParentDirectories)
+		require.NoError(t, err)
+	}
+
+	// Read entries
+	entries, err := wf.ReadDir(ctx, ".")
+	require.NoError(t, err)
+	assert.Len(t, entries, len(files))
+	names := []string{}
+	for _, e := range entries {
+		names = append(names, e.Name())
+	}
+	slices.Sort(names)
```

Comment:
This should already be sorted by the filer (also see the workspace files impl of this function).

Reply:
Done